shared/logic/Update.ts

```diff
@@ -269,6 +270,15 @@ function tickTile(xy: Tile, gs: GameState, offline: boolean): void {
    const inputWorkerCapacity = totalMultiplierFor(xy, "worker", 1, gs);
    const { total, used } = getStorageFor(xy, gs);
 
+   const options = getGameOptions();
+
+   if (total > 0) {
```

Comment:
Might be a better way of checking if a building has storage or not.

Reply:
I think this is probably the best way. BuildingStorageComponent also check `Number.isFinite(total)`, although from reading the code, I don't think `getStorageFor` would ever return Infinite.

https://github.com/fishpondstudio/CivIdle/blob/d0b063b9a1764d1051479e2c0bbad8e4bf48a2c4/src/scripts/ui/BuildingStorageComponent.tsx#L18

It may be worth it to extract the logic to a function called `buildingHasStorage` so that the logic is centralized in one place.

Reply:
Moved logic to a new method in BuildingLogic.ts